astro/src/content/blog/fine-grained-authorization.mdx

```diff
@@ -320,6 +321,8 @@ Fine-grained authorization doesn't just mean more rules-it means building a smar
 
 That's why you need Policy Decision Points (PDPs). 
 
+> _Ready to upgrade your access model? [Book a demo](/contact) to learn how FusionAuth powers dynamic, fine-grained authorization across modern stacks._
```

Comment:
uh, I wouldn't make this claim.

Reply:
you could say something like ... `learn how FusionAuth can enable dynamic, fine-grained authorization across modern stack.` but even then we'd want to be clear you'd be partnering with permit or cerbos.